coral/.husky/pre-commit

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+STAGED_FILES=$(git diff --staged --name-only)
+
+if echo "$STAGED_FILES" | grep -q coral;
+then
+  echo 'Files on coral staged'
```

Comment:
I assume this echo was used for the development, but not needed anymore?

Reply:
Yes. Sorry 😞  Seems like I should have checked my changes better before opening a PR.